pages/deposit/[symbol].tsx

```diff
@@ -144,7 +144,7 @@ const Deposit: NextPage<
                     <ClientComponent>
                       <Slider
                         key={'deposit-slider'}
-                        count={proceedsByEpochsDeposited.length}
+                        length={proceedsByEpochsDeposited.length}
```

Comment:
```suggestion
                        length={proceedsByEpochsDeposited?.length || 0}
```

Reply:
Since `initialData: []`, proceedsByEpochsDeposited is never null

Reply:
I'll do so in a separate PR